test/test_mcmc.py

```diff
@@ -135,7 +136,6 @@ def model(data):
     tau_values, counts = onp.unique(tau_posterior, return_counts=True)
     mode_ind = np.argmax(counts)
     mode = tau_values[mode_ind]
-    assert max(tau_values) == 44
```

Comment:
I verified that the mode remains the same when we increase the number of samples. This assertion however is violated and I don't think there is a reason why this should be true, so I'm removing it.

Reply:
Yup, you are right. I don't think so too. :D